test/e2e/simple_preemptor/simple_preemptor_test.go

```diff
@@ -79,13 +81,29 @@ var _ = ginkgo.BeforeSuite(func() {
 
 	// Extract node allocatable resources
 	for _, node := range nodes.Items {
-		if node.Name == Worker1 {
+		// skip master if it's marked as such
+		node := node
+		if k8s.IsMasterNode(&node) || !k8s.IsComputeNode(&node) {
```

Comment:
Isn't any node which is not a master node by definition a compute node? Why the additional check?

Reply:
I have seen 3 types - 1. Master 2. Workers 2a). Infra 2b). Compute. We have to consider only compute nodes, not even 2a for running pods. Once Compute nodes has been chosen, we have to limit the no. of compute nodes (mostly two) for running pods and taint other compute nodes as un schedulable based on the use cases.